test/views/certificate/VatCertificateViewSpec.scala

```diff
@@ -96,47 +93,26 @@ class VatCertificateViewSpec extends ViewBaseSpec {
         }
         "contains the trading name" in {
           val regDateRow = card.select(".govuk-check-your-answers:nth-of-type(2)")
-          val date = LocalDate.now().minus(Period.ofDays(2))
           regDateRow.select("dt").text() shouldBe "Trading name"
           regDateRow.select("dd").text() shouldBe s"ABC Traders"
         }
         "contains the business type" in {
           val certDateRow = card.select(".govuk-check-your-answers:nth-of-type(3)")
-          val date = LocalDate.now()
           certDateRow.select("dt").text() shouldBe "Business type"
           certDateRow.select("dd").text() shouldBe s"Organisation"
         }
         "contains the trade classification" in {
           val certDateRow = card.select(".govuk-check-your-answers:nth-of-type(4)")
-          val date = LocalDate.now()
           certDateRow.select("dt").text() shouldBe "Trade classification (SIC code)"
           certDateRow.select("dd").text() shouldBe s"6602"
         }
         "contains the address" in {
           val certDateRow = card.select(".govuk-check-your-answers:nth-of-type(5)")
-          val date = LocalDate.now()
           certDateRow.select("dt").text() shouldBe "Principal place of business address"
           certDateRow.select("dd").text() shouldBe s"Line 1 Line 2 TF4 3ER"
         }
       }
 
-      "have the banking details card" that {
```

Comment:
Eight of the tests in this View Spec are now failing due to the banking section being removed but the CSS selectors not being updated.